bot/openai_helper.py

```diff
@@ -28,7 +28,8 @@
 GPT_4_32K_MODELS = ("gpt-4-32k", "gpt-4-32k-0314", "gpt-4-32k-0613")
 GPT_4_VISION_MODELS = ("gpt-4-vision-preview",)
 GPT_4_128K_MODELS = ("gpt-4-1106-preview","gpt-4-0125-preview","gpt-4-turbo-preview")
-GPT_ALL_MODELS = GPT_3_MODELS + GPT_3_16K_MODELS + GPT_4_MODELS + GPT_4_32K_MODELS + GPT_4_VISION_MODELS + GPT_4_128K_MODELS
+GPT_4O_MODELS = ("gpt-4o")
```

Comment:
You missed the comma to create a tuple with one item
```
TypeError: can only concatenate tuple (not "str") to tuple
Traceback (most recent call last):
  File "/app/bot/main.py", line 7, in <module>
    from openai_helper import OpenAIHelper, default_max_tokens, are_functions_available
  File "/app/bot/openai_helper.py", line 32, in <module>
    GPT_ALL_MODELS = GPT_3_MODELS + GPT_3_16K_MODELS + GPT_4_MODELS + GPT_4_32K_MODELS + GPT_4_VISION_MODELS + GPT_4_128K_MODELS + GPT_4O_MODELS
```

```suggestion
GPT_4O_MODELS = ("gpt-4o",)
```